pkg/tablebackup/tablebackup.go

```diff
@@ -477,7 +503,13 @@ func (t *TableBackup) hasRows() (bool, error) {
 func (t *TableBackup) Truncate() error {
```

Comment:
It looks like this method is not used anymore.

Reply:
removed